app/views/users/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Here is a list of posts for all user</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+ 
+    </style>
+  </head>
+
+  <body>
+    <h1>Here is a list of Users<h1/>
+      <ul>
+      <% @users.each do |user| %>
+        <li style="cursor: pointer;" >
+        <a href='users/<%= user.id %>'>
+        <%= user.name %>
+        <img  src="<%= user.photo %>" alt="Avatar" />
+        </a>
+        <% if user.postscounter > 0 %>
+        <p>Posts: <%= user.postscounter %></p>
+        <p><%= user.photo %></p>
+        <% end %>
+        <% end %>
+        </li>
+    </ul>
+  </body>
+</html>
```

Comment:
- [ ] Kindly ensure your User index page matches the design.

  Your Implementation | Wireframe
  :---:|:---:
  ![irorrr](https://user-images.githubusercontent.com/36542986/225568519-1ba107aa-f91e-4095-9c9c-a2ca5f4685b6.jpg) | ![prprr](https://user-images.githubusercontent.com/36542986/225568509-6feb7d6b-3f91-4fbe-941f-af97c38cfdf5.jpg)